build.cmd

```diff
@@ -587,6 +587,7 @@ echo skipmscorlib: skip building System.Private.CoreLib ^(default: System.Privat
 echo skipnative: skip building native components ^(default: native components are built^).
 echo skiptests: skip building tests ^(default: tests are built^).
 echo skipbuildpackages: skip building nuget packages ^(default: packages are built^).
+echo buildstandalone: builds the GC in a standalone mode.
```

Comment:
you're right, will do - the command is actually "buildstandalonegc", I think I just typo'd it here.